cloudbuild.deploy.yaml

```diff
@@ -1,54 +1,54 @@
 steps:
+  - id: package_js
+    name: "node"
+    entrypoint: /bin/bash
+    args:
+      - -c
+      - |
+        set -x
+        ./run_test.sh -lbc
 
-- id: package_js
-  name: 'node'
-  entrypoint: /bin/bash
-  args:
-  - -c
-  - |
-    set -x
-    ./run_test.sh -lbc
+  - id: flask_test
+    name: python:3.7-slim
+    entrypoint: /bin/sh
+    args:
+      - -c
+      - |
+        set -x
+        ./run_test.sh -p
 
-- id: flask_test
-  name: python:3.7-slim
-  entrypoint: /bin/sh
-  args:
-  - -c
-  - |
-    set -x
-    ./run_test.sh -p
+  - id: deploy
+    name: "gcr.io/cloud-builders/gcloud"
+    entrypoint: /bin/bash
+    args:
+      - -c
+      - |
+        set -x
+        cd server
+        gcloud config set project datcom-browser-staging
+        # bdist_wheel for grpcio takes a long time, seeting timeout to be 1200s
```

Comment:
nit: s/seeting/setting

what happens if we hit the timeout? should we continue with the deployment?

Reply:
Then cloudbuild would error out. Tried locally it the gcloud build takes < 10min, so this should be safe. Will need to watch for the first few times deployment

Reply:
ok, let's keep an eye on it